### PR TITLE
doc multi-tenancy

### LINKED ISSUE #42: Doc multi-tenancy
Dev PR: https://github.com/ConsenSys/tessera/pull/1190

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull Request Description

Add page on Tessera/GoQuorum multi-tenancy, linking to existing GoQ content.

## Fixed Issue(s)

fixes #42 

## Preview

https://pegasys-tessera--122.com.readthedocs.build/en/122/Concepts/Multitenancy/

### CODE REVIEW

File: docs/Concepts/Multitenancy.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The public state remains publicly available to all tenants, and private states are logically separated.
+
+For more information about multi-tenancy and MPS, see the [GoQuorum documentation](https://consensys.net/docs/goquorum/en/latest/concepts/multi-tenancy/).
+
+## Using multi-tenancy via multiple private states
+
+To use multi-tenancy via multiple private states, you must have:
+
+- Tessera version `21.4.0` or later installed.
+- GoQuorum version `21.4.2` or later installed.
```
Comment (taccatisid) [inline docs/Concepts/Multitenancy.md:27]: Would it make sense to mention Besu as an alternative to GoQuorum here?

File: docs/Concepts/Multitenancy.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Multi-tenancy via multiple private states (MPS) allows multiple tenants to use the same GoQuorum node, with each
+tenant having its own private state(s).
+Each tenant can perform all operations (create, read, and write) on any contract in its private state, and a single
+tenant can have access to multiple private states.
+Multi-tenancy provides a user experience similar to a user running their own managed node.
+
+The public state remains publicly available to all tenants, and private states are logically separated.
+
+For more information about multi-tenancy and MPS, see the [GoQuorum documentation](https://consensys.net/docs/goquorum/en/latest/concepts/multi-tenancy/).
```
Comment (macfarla) [inline docs/Concepts/Multitenancy.md:20]: would be better to link to stable not latest

File: docs/Concepts/Multitenancy.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    As described in the migration guide, you must run the [Tessera multi-tenancy migration tool](../HowTo/Migrate/Migration-Multitenancy.md)
+    to upgrade Tessera to support multi-tenancy.
+
+Configure multi-tenancy via multiple private states according to the [GoQuorum multi-tenancy guide](https://consensys.net/docs/goquorum/en/stable/configure-and-manage/manage/multi-tenancy/multi-tenancy/).
+
+!!! note
+
+    As described in the multi-tenancy guide, you must [configure MPS in Tessera](../HowTo/Configure/Multiple-private-state.md)
+    to use multi-tenancy.
+
```
Comment (macfarla) [inline docs/Concepts/Multitenancy.md:42]: ^ the above "important" and "note" are a bit confusing. Does it boil down to 2 things - firstly, if you're running earlier version/s, you need to upgrade. And secondly, you need to do config in _both_ Tessera and GoQuorum to make it work?

File: docs/Concepts/Multitenancy.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+description: Tessera and GoQuorum multi-tenancy
+---
+
+# Multi-tenancy via multiple private states
+
+In a typical [GoQuorum](https://consensys.net/docs/goquorum/en/stable/) network, each participant (tenant) uses its own
```
Comment (macfarla) [inline docs/Concepts/Multitenancy.md:7]: their own

**Review by macfarla — APPROVED** [review]

File: docs/Concepts/Multitenancy.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    - Tessera version `21.4.0` or later installed.
+    - GoQuorum version `21.4.2` or later installed.
+
+    If running an earlier GoQuorum or Tessera version, upgrade your existing nodes to enable MPS and multi-tenancy using
+    the [GoQuorum MPS migration guide](https://consensys.net/docs/goquorum/en/stable/configure-and-manage/manage/multi-tenancy/migration/).
+    As described in the guide, this includes running the [Tessera multi-tenancy migration tool](../HowTo/Migrate/Migration-Multitenancy.md)
+    to upgrade Tessera to support multi-tenancy.
+
+2. Configure multi-tenancy via MPS according to the [GoQuorum multi-tenancy guide](https://consensys.net/docs/goquorum/en/stable/configure-and-manage/manage/multi-tenancy/multi-tenancy/).
+   As described in the guide, this includes [configuring MPS in Tessera](../HowTo/Configure/Multiple-private-state.md).
```
Comment (bgravenorst) [inline docs/Concepts/Multitenancy.md:40]:
```suggestion
2. [Configure multi-tenancy via MPS](https://consensys.net/docs/goquorum/en/stable/configure-and-manage/manage/multi-tenancy/multi-tenancy/), this includes [configuring MPS in Tessera](../HowTo/Configure/Multiple-private-state.md).
```

File: docs/Concepts/Multitenancy.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+    - Tessera version `21.4.0` or later installed.
+    - GoQuorum version `21.4.2` or later installed.
+
+    If running an earlier GoQuorum or Tessera version, upgrade your existing nodes to enable MPS and multi-tenancy using
+    the [GoQuorum MPS migration guide](https://consensys.net/docs/goquorum/en/stable/configure-and-manage/manage/multi-tenancy/migration/).
+    As described in the guide, this includes running the [Tessera multi-tenancy migration tool](../HowTo/Migrate/Migration-Multitenancy.md)
+    to upgrade Tessera to support multi-tenancy.
+
+2. Configure multi-tenancy via MPS according to the [GoQuorum multi-tenancy guide](https://consensys.net/docs/goquorum/en/stable/configure-and-manage/manage/multi-tenancy/multi-tenancy/).
```
Comment (bgravenorst) [inline docs/Concepts/Multitenancy.md:39]: Normally we can omit "en/stable" from the urls, but now it seems to 404. @NicolasMassart, do you have any idea why?

File: docs/Concepts/Multitenancy.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+   As described in the guide, this includes [configuring MPS in Tessera](../HowTo/Configure/Multiple-private-state.md).
+
+3. You can also follow the multi-tenancy guide to add a new tenant to a multi-tenant node.
```
Comment (bgravenorst) [inline docs/Concepts/Multitenancy.md:42]:
```suggestion
3. You can also [add a new tenant to a multi-tenant node](https://consensys.net/docs/goquorum/en/stable/configure-and-manage/manage/multi-tenancy/multi-tenancy/#add-a-new-tenant-to-multi-tenant-node).
```

File: docs/HowTo/Configure/Multiple-private-state.md [hunk 8]
```diff
@@ -4,9 +4,11 @@ description: Configure Tessera to support GoQuorum multiple private states featu
 
 # Configure multiple private states
 
-You can enable support for [GoQuorum multiple private states (MPS)](https://docs.goquorum.consensys.net/en/latest/Concepts/Multitenancy/#multiple-private-states)
-by setting [`enableMultiplePrivateStates`](../../Reference/SampleConfiguration.md#features) in the Tessera
-[configuration file](Tessera.md) to `true`.
+Using [multi-tenancy](../../Concepts/Multitenancy.md) requires enabling support for
```
Comment (bgravenorst) [inline docs/HowTo/Configure/Multiple-private-state.md:7]: To use multi-tenancy, enable GoQuorum multiple private states (MPS)

File: docs/HowTo/Configure/Multiple-private-state.md [hunk 9]
```diff
@@ -4,9 +4,11 @@ description: Configure Tessera to support GoQuorum multiple private states featu
...
+[GoQuorum multiple private states (MPS)](https://docs.goquorum.consensys.net/en/latest/Concepts/Multitenancy/#multiple-private-states).
+
+You can do this by setting [`enableMultiplePrivateStates`](../../Reference/SampleConfiguration.md#features) in the
```
Comment (bgravenorst) [inline docs/HowTo/Configure/Multiple-private-state.md:10]: To do this, set the...

**Review by bgravenorst — APPROVED** [review]
Some feedback. Feel free to merge once approved/rejected.

**Pull request merged**